packages/wallet-adapter-core/src/WalletCore.ts

```diff
@@ -371,23 +374,46 @@ export class WalletCore extends EventEmitter<WalletCoreEvents> {
     }
   }
 
+  /**
+   * This function is for signing and submitting a transaction using the `@aptos-labs/ts-sdk` (aka the v2 SDK)
+   * input types. It's internally converting the input types to the old SDK input types and then calling
+   * the v1 SDK's `signAndSubmitBCSTransaction` with it.
+   * 
+   * @param transactionInput the transaction input
+   * @param options max_gas_amount and gas_unit_limit
+   * @returns the response from the wallet's signAndSubmitBCSTransaction function
+   */
+  async submitTransaction(
+    transactionInput: InputGenerateTransactionData,
+    options?: TransactionOptions,
+  ): Promise<any> {
+    const payloadData = transactionInput.data;
+    const aptosConfig = new AptosConfig({network: convertNetwork(this._network)});
+    // TODO: Refactor this any, and remove the need for it by fixing the if ("bytecode" in data) stuff in `generateTransaction` in the v2 SDK
```

Comment:
can we add this task into our roadmap so we wont forget about it?

Reply:
added it!